docs/book/orders.rst

```diff
@@ -116,6 +116,58 @@ Order has also its own state, which can have the following values:
 
     The state machine of order is an obvious extension to the :doc:`state machine of checkout </book/checkout>`.
 
+Shipments of an Order
+---------------------
+
+An **Order** in Sylius holds a collection of Shipments on it. Each shipment in that collection has its own shipping method and has its own state machine.
+This lets you to divide an order into several different shipments (like sending physical objects via DHL and sending a link to downloadable files via e-mail).
```

Comment:
``...a link to files downloadable via e-mail`` - this would mean that files can be downloaded via e-mail :)